code/src/components/AlbumList/AlbumList.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Album } from '../Album/Album';
+
+export const AlbumList = (props) => {
+  console.log(props)
```

Comment:
Good thing that you console.logged the props. Maybe you can remove the console.log when you know it works? (I don't know if they care if we still have it there.)